pintracker/statelesstracker/statelesstracker.go

```diff
@@ -0,0 +1,380 @@
+package statelesstracker
+
+import (
+	"context"
+	"errors"
+	"sync"
+
+	"github.com/ipfs/ipfs-cluster/api"
+	"github.com/ipfs/ipfs-cluster/optracker"
+	"github.com/ipfs/ipfs-cluster/pintracker/ptutil"
+
+	rpc "github.com/hsanjuan/go-libp2p-gorpc"
+	cid "github.com/ipfs/go-cid"
+	logging "github.com/ipfs/go-log"
+	peer "github.com/libp2p/go-libp2p-peer"
+)
+
+var logger = logging.Logger("pintracker")
+
+// StatelessPinTracker uses the optracker.OperationTracker to manage
+// transitioning shared ipfs-cluster state (Pins) to the local IPFS node.
+type StatelessPinTracker struct {
```

Comment:
If we rename the module to `stateless` and this to `Tracker` things sound good: `stateless.Tracker`.